src/App.js

```diff
@@ -1,3 +1,19 @@
+import CropItem from './components/CropItem'
+import crops from './data.json'
+
 export default function App() {
-  return <div className="App"></div>
+  return (
+    <div>
+      {crops.map(crop => {
+        const { id, attributes } = crop
+        return (
+          <CropItem
+            key={id}
+            name={attributes.name}
+            image={attributes.main_image_path}
```

Comment:
note for later

Reply:
The data.json is a selfmade compilation of data from an API. I will probably fetch the original API later at some point so I thought it would be best leaving the names as they are.. Would you change them?